client/src/App.js

```diff
@@ -27,7 +35,7 @@ function App() {
           {user.userType === "doctor" ? <DoctorForm /> : <PatientForm />}
         </Route>
         <Route path="/register">
-          <Registration />
+          {!currentUser ? <Registration /> : <Redirect to="/"/>}
         </Route>
```

Comment:
Redirect to login will be better. Ideally with a message but that can come afterwards.